medical_medicament_attribute/views/medical_medicament_attribute_type_view.xml

```diff
@@ -3,7 +3,7 @@
 <!--
     Copyright (C) 2016 LasLabs [https://laslabs.com]
     @author Dave Lasley <dave@laslabs.com>
-    The license is in the file __openerp__.py
+    The license is in the file __odoo__.py
```

Comment:
The v10 equivalent of ``__openerp__.py`` is ``__manifest__.py``. However, if you're gonna update these at all, I would instead suggest just switching over to the latest OCA copyright format, which can be found [here](https://github.com/OCA/maintainer-tools/blob/master/template/module/data/model_name_data.xml#L2). The module is licensed under LGPL-3.0.